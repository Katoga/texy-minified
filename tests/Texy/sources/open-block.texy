/--code php
<?php
if (strpos($_SERVER['REQUEST_URI'], 'controller.php')) $_SERVER['REQUEST_URI'] = '/cs/try/';

[syntax]: syntax Syntaxe .{font-weight:bold}
