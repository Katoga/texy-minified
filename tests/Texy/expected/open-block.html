<pre class="php"><code>&lt;?php
if (strpos($_SERVER['REQUEST_URI'], 'controller.php')) $_SERVER['REQUEST_URI'] = '/cs/try/';</code></pre>
